lib/google.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+/* global JuttleAdapterAPI */
+let _ = require('underscore');
+let logger = JuttleAdapterAPI.getLogger('google-analytics');
+let JuttleMoment = JuttleAdapterAPI.types.JuttleMoment;
+let Promise = require('bluebird');
+let google = require('googleapis');
+let analytics = google.analytics('v3');
+
+let auth, authorized, accountId;
+
+function init(config) {
+    auth = new google.auth.JWT(
```

Comment:
At this point can you check that the config has a 'service_account' property and 'client_email'/'private_key' subproperties? I accidently used 'service-account' and the error was pretty ugly.